apps/communications/contacts/js/activities.js

```diff
@@ -96,12 +96,8 @@ var ActivityHandler = {
         break;
       case 'open':
         if (this.isvCardActivity(activity)) {
-          var self = this;
-          var dependency =
-                         '/shared/js/contacts/import/utilities/vcard_reader.js';
-          LazyLoader.load(dependency, function() {
-            self.getvCardReader(activity.source.data.blob,
-                                           self.readvCard.bind(self, activity));
+          LazyLoader.load(['/contacts/js/activities_vcard.js'], () => {
```

Comment:
nit: if only one script file is loaded then it is only necessary to pass a string